roles/load_aware_undeploy_trimaran/tasks/main.yml

```diff
@@ -0,0 +1,11 @@
+- name: Delete Trimaran namespace
+  shell:
+    oc delete ns trimaran
```

Comment:
this role is quite fine (except the task below),
but what I would prefer is:
in `roles/load_aware_undeploy_trimaran/files/`, create a symlink to `roles/load_aware_deploy_trimaran/files/trimaran.yaml`
and in this file, just `oc delete -f {{ file }}`
this way, you know that no matter what you add to the file above, it will be properly deleted

at least I hope it will work, as the namespace is first in the yaml file 🤔

Reply:
This feels much safer. I adopted this in the most recent commit! I have a variable in vars/main/resources.yml pointing to the deploy_trimaran configuration files rather than a symlink though. Not sure if there was a reason for one over the other.

Reply:
> I have a variable in vars/main/resources.yml pointing to the deploy_trimaran configuration files rather than a symlink though. 

yes, works equally. One over the other is mostly a matter of taste,
my idea is to keep the role "standalone", so at the Ansible level, it's not referencing anything outside of the role. 
but at the FS level it is, so the encapsulation isn't complete anyway ....